docs/.vuepress/config.js

```diff
@@ -50,51 +50,51 @@ module.exports = {
         sidebar: [
           '/install/',
           {
-            title: 'Essentials',
-            path: '/essentials/',
+            title: 'Concepts',
+            path: '/concepts/',
             children: [
               {
-                title: 'IPFS in a nutshell',
+                title: 'IPFS 101',
                 sidebarDepth: 1,
                 collapsable: false,
                 children: [
-                  '/essentials/what-is-ipfs',
-                  '/essentials/how-ipfs-works',
-                  '/essentials/usage-ideas-examples',
-                  '/essentials/glossary',
-                  '/essentials/faq'
+                  '/concepts/what-is-ipfs',
+                  '/concepts/how-ipfs-works',
+                  '/concepts/usage-ideas-examples',
+                  '/concepts/glossary',
+                  '/concepts/faq'
                 ]
               },
               {
                 title: 'Content-addressed storage',
```

Comment:
```suggestion
                title: 'Content-addressed data',
```
I remember @hugomrdias suggesting that data would be more appropriate here. What do you think?

Reply:
I'm down. That does make more sense.